templates/zerver/help/mute-a-topic.md

```diff
@@ -38,15 +38,33 @@ Follow the same procedure to unmute the topic.
 
 Follow the same procedure to unmute the topic.
 
+### From the slashed bell icon
+
+{start_tabs}
+
+1. Go to the **Recent topics** or find the recipient bar of the topic that you wish
+   to mute.
```

Comment:
I'm not sure `recipient bar` is something users would know about. But I'm not sure what else to use either. What do you think?

Reply:
Yeah, agreed. If we don't have another proper term, then I think we should attach an example picture of the `recipient bar`.